radicle-node/src/service.rs

```diff
@@ -777,6 +836,25 @@ where
         // TODO
     }
 
+    fn disconnect_unresponsive_peers(&mut self, now: LocalTime) {
+        let stale = self
+            .sessions
+            .negotiated()
+            .filter(|(_, session)| STALE_CONNECTION_TIMEOUT <= now - session.last_active);
```

Comment:
Usually we put the variable on the left side of the comparison, since it is the "subject"